src/server/bg_services/server_monitor.js

```diff
@@ -75,16 +73,8 @@ async function run_monitors() {
     await _check_internal_ips();
     await _check_disk_space();
 
-    if (PLATFORM !== 'docker') {
-        await _verify_ntp_cluster_config();
-        await _verify_proxy_cluster_config();
```

Comment:
Good change, but what will we see in the FE on the server page now we don't verify these statuses? We don't have to change the FE now, but may be set to always green until we do remove it?

Reply:
currently on the UI, it is shown as Not configured: 
![NooBaa_Management_Console](https://user-images.githubusercontent.com/28217223/59031276-e2d50400-886b-11e9-88ca-97eacf8753d3.png)